src/contract_utils/lido_keys.py

```diff
@@ -0,0 +1,58 @@
+import binascii
+from functools import lru_cache
+from typing import List, Dict
+
+from hexbytes import HexBytes
+from lido_sdk import Lido
+from lido_sdk.methods.typing import OperatorKey
+from web3 import Web3
+
+from src.providers.beacon import BeaconChainClient
+from src.web3_utils.typings import MergedLidoValidator, ModifiedOperatorKey, ModifiedOperator
+
+
+def _get_dict_of_keys(validators: List[OperatorKey]) -> Dict[str, OperatorKey]:
+    """Key is pubkey in hex format. Value is data from key registry."""
+    return {
+        '0x' + binascii.hexlify(validator['key']).decode(): validator for validator in validators
+    }
+
+
+@lru_cache(maxsize=2)
+def get_lido_node_operators(w3: Web3, block_hash: HexBytes) -> List[ModifiedOperator]:
```

Comment:
I pos this code about lido validator, not the operator. Because of the - node operator, it's a human=)